### ISSUE TITLE
FreeSWITCH doesn't create a signed Identity header.

### ISSUE THREAD

**Issue by ynasida** [signalwire/freeswitch#2000]
Hello,

 I am trying to  create a signed Identity header using Freeswitch. I have all stir-shaken params in sofia.conf according instructions (https://developer.signalwire.com/freeswitch/FreeSWITCH-Explained/Configuration/Sofia-SIP-Stack/STIR-SHAKEN_75300891/).
I also have set channel variable sip_stir_shaken_attest=A , but... nothing happens.
Freeswitch just adds this new variable and does nothing. 
If I try to use app sofia_stir_shaken_vs - it tries to verify Identity but I do not see any Freeswitch activity when I try to sign the call.

looking at source code i see:
`if ((stir_shaken_attest = switch_channel_get_variable(tech_pvt->channel, "sip_stir_shaken_attest"))) {`

I am really not sure why FS ignores sip_stir_shaken_attest variable i set.

FreeSWITCH Version 1.10.9-release-21-a615e85afc~64bit (-release-21-a615e85afc 64bit)

**To Reproduce**
        <action application="set" data="sip_stir_shaken_attest=A"/>
and bridge

2023-03-22 08:28:10.379904 100.00% [DEBUG] mod_dptools.c:1673 SET sofia/internal/username@sip.domain.com [stir_shaken_attest]=[A]

**Comment by ynasida** [comment]
Well. FS should be installed from source to get it working. FYI. Probably saved a lot of time for somebody.

**Comment by bhavik6842** [comment]
I have also installed freeswitch with "apt" and facing same issue.
Are you sure after installing FS from Source , Will it work ?

**Comment by ynasida** [comment]
Yes I had it working with FS installed from source. But for me it is not the option because I do not want to re-install a lot of FS nodes.

**Comment by bhavik6842** [comment]
Is there any good documentation for implementation stir/shaken with freeswitch ?
Suggestion is also fine like, is it mandatory to get any kind of Token varification service OR freeswitch can handle all within just use dialplan and certificate ?

**Comment by ynasida** [comment]
@bhavik6842 did you try with FS installed from source ? Like I mentioned before it was the only way to sign outbound call using Freeswitch. At the moment,  I use kamailio for things like this.

**Comment by bhavik6842** [comment]
Yes i tried with fresh installation with source. But it always return above output.
Do i need to install libstirshaken signalwire liberary ?
But how to use it with freeswitch, they give us command but where i need to run those command on call ?

**Comment by bhavik6842** [comment]
Great,So I already installed libstirshaken but don't know how to push identity header.
Also do we require any Caller-id authenticators Service provider to validate Caller-id ?

Sorry, very less documentation and so many questions.
